core/src/test/scala/za/co/absa/fadb/status/handling/implementations/StandardStatusHandlingTest.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.fadb.status.handling.implementations
+
+import org.scalatest.funsuite.AnyFunSuiteLike
+import org.scalatest.matchers.should.Matchers.{a, convertToAnyShouldWrapper}
```

Comment:
I didn't know about this `a` but I don't think it's used

Reply:
correct, it's not used anymore in the test cases, removed it